src/eos-crash-metrics.c

```diff
@@ -181,13 +248,18 @@ main (int argc, char **argv)
 
   url = get_eos_ostree_repo_url (repo);
   commit = get_eos_ostree_deployment_commit (sysroot, repo);
-
   if (!url || !commit)
     {
       g_warning ("Unable to get OSTree url or commit, perhaps the system has been tampered with?");
       return EXIT_FAILURE;
     }
 
+  if (g_str_has_prefix (path, "/app/bin"))
+    {
+      g_message ("%s is likely a Flatpak", path);
+      g_warning ("Flatpak name: %s", get_flatpak_application_name (path, &error));
+    }
+
   report_crash (path, signal, timestamp, commit, url);
```

Comment:
I think as part of this story we also wanted to report the flatpak ref as part of the crash metric, can you pass that through here?

Reply:
I think we also wanted to figure out which runtimes were linked to the app as well, though that can be done in a followup